clients/web/src/views/body/FolderView.js

```diff
@@ -19,7 +19,12 @@ var FolderView = View.extend({
         this.folderEdit = settings.folderEdit || false;
         this.itemCreate = settings.itemCreate || false;
 
+        this.render();
+    },
+
+    render: function () {
         this.hierarchyWidget = new HierarchyWidget({
+            el: this.$el,
```

Comment:
In theory, I see nothing wrong with it. To me, it should just naturally force us to consider whether we need the parent view, or whether the child view could be more directly instantiated via the router. In many cases, there is supplemental state, logic, or sub-sub-views that need to be added by the parent class. In this case, `FolderView` could probably be elided entirely, but that's obviously outside the scope of this PR.